packages/orbit-components/src/Dialog/index.js

```diff
@@ -184,6 +185,34 @@ const Dialog = ({
         </StyledDialogCenterWrapper>
       </StyledDialog>
     </Portal>
+  ) : (
+    <StyledDialog
+      data-test={dataTest}
+      shown={shown}
+      onClick={handleClose}
+      tabIndex="0"
+      role="dialog"
+      aria-modal="true"
+      aria-labelledby={dialogID}
+    >
+      <StyledDialogCenterWrapper>
+        <StyledDialogContent shown={shown} ref={ref} id={dialogID}>
+          {illustration && <IllustrationContainer>{illustration}</IllustrationContainer>}
+          <Stack spacing="XSmall" spaceAfter="medium">
+            {title && <Heading type="title3">{title}</Heading>}
+            {description && <Text type="secondary">{description}</Text>}
+          </Stack>
+          <Stack
+            direction="column-reverse"
+            spacing="XSmall"
+            largeMobile={{ direction: "row", justify: "end" }}
+          >
+            {secondaryAction && <StyledAction>{secondaryAction}</StyledAction>}
+            <StyledAction>{primaryAction}</StyledAction>
+          </Stack>
+        </StyledDialogContent>
+      </StyledDialogCenterWrapper>
+    </StyledDialog>
```

Comment:
If all of this added code is the same as in the portal, don't copy this here. You can do something like this:

```jsx
const dialog = (
  // JSX code
)

return insidePortal ? (
  <Portal renderInto="modals">
    {dialog}
  </Portal>
) : dialog;
```

Same suggestion for other components.